POEApi.Model/JSONProxy/Stash.cs

```diff
@@ -26,6 +26,14 @@ public class Requirement
         public int DisplayMode { get; set; }
     }
 
+    public class IncubatedDetails
+    {
+        public string Name { get; set; }
```

Comment:
We should have comments explaining exactly what these values mean. (E.g., how `Progress` and `Total` relate isn't intuitively obvious.)

Reply:
These are tied to the properties that come over the wire on the JSon objects.

Reply:
One of the big mistakes in Procurement was tying its internal models to the json ones that come from the API, they should be separated and mapped across. It would have saved us so many times in the past with so many issues.